package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "corewar",
-  "version": "0.0.29",
+  "version": "0.0.45",
```

Comment:
I think you can build your npm package locally with `npm prepublishOnly` and add a file reference for the output into the consuming app's `package.json` - saves you having to reversion and publish over and over when working on an issue...